reportportal/values.yaml

```diff
@@ -5,6 +5,18 @@
 ## String to fully override reportportal.fullname template
 ##
 # fullnameOverride:
+extraInitContainers: {}
```

Comment:
Please put that in the end of the file and add a comment section right above the field.

Reply:
done : https://github.com/reportportal/kubernetes/pull/233/files#diff-c554617a65c1ad51602ac8780bbe79db48ddcc7b00106d755361a16301b571ec